### ISSUE TITLE
Validate input field

### ISSUE THREAD

**Issue by funnybones** [vitalets/angular-xeditable#416]
Hello, is there a way to validate the input field created, change it to numbers or apply ng-patterns ?

Thank you,
Alex.

**Comment by nailaandleeb** [comment]
i want this validation too......so is there any solution???

**Comment by funnybones** [comment]
Yes, replace this inline edit with an input box that has no borders, there are many angular alternatives for this (e.g. http://www.tamas.io/inline-editing-with-angularjs/). Basically you have to use a normal input and do the validation in pure angular style on that input. 

Let me know any questions,
Alex.

**Comment by ckosloski** [comment]
You can use a `input type="number"` for number input.

You could try the html5 `pattern` attribute for applying patterns.

**Comment by ckosloski** [comment]
@fnbns can this issue be closed?
